### PR TITLE
Feat/minorchanges

### PR DESCRIPTION
**Pull request by tsvetelinak0**
### What change does this PR introduce?
title section in about_us.html and constact_us.html fixed styles
django version changed in travel.yml from 2.0.8 to 2.2.10 as per requirements.txt

### Notes
changes might not be visible but they will be in help in future when pages grow

### Ticket
No ticket for that

### Screenshots
Screenshot 2020-05-31 at 09 58 28

### CODE REVIEW

File: src/styles/styles.scss [hunk 1]
```diff
@@ -37,6 +36,62 @@ html, body {
   height: 100%;
 }
 
+.index_bottom_container {
```
Comment (lili2311) [inline src/styles/styles.scss]: please can you try to keep PRs tidy and not also commit changes from other branches? Could you try to remove any changes in this PR that are already in some other PR?

File: src/styles/components/_title.scss [hunk 2]
```diff
@@ -7,9 +7,16 @@
 .homepage_section_block {
 	align-items: center;
     height: 450px;
+<<<<<<< HEAD
```
Comment (lili2311) [inline src/styles/components/_title.scss]: please fix the merge conflict here?

Comment (tsvetelinak0) [reply src/styles/components/_title.scss]: that doesn't appear anymore when I redone the previous PR. what should I do?

Comment (tsvetelinak0) [reply src/styles/components/_title.scss]: also don't know what I am messing as didn't merge anything and it merged and closed the PR :(

**Pull request closed without merging**